src/test/librados/tier_cxx.cc

```diff
@@ -3600,6 +3602,854 @@ TEST_F(LibRadosTwoPoolsPP, ManifestFlushRead) {
   cluster.wait_for_latest_osdmap();
 }
 
+TEST_F(LibRadosTwoPoolsPP, ManifestSnapRefcount) {
```

Comment:
I don't want to block the merge on this, but in the future, please make an effort to factor out helper methods so that reviewers and future readers can quickly understand how your tests work.

Reply:
I got it.